index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Microverse-Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <div class="con1">
+        <h1 class="logo-title">My Logo</h1>
+        <nav>
+          <img class="ham-img" src="/images/ham-menu.png" alt="Menu">
+        </nav>
+      </div>
+    </header>
+
+
+    <section class="Homepage">
+      <h2 class="hero-title">I’m Steve
+        Glad to see you!</h2>
+      <p class="hero-description">I’m a software developer! I can help you build a product, feature or website. Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hesitate to contact me.</p>
+      <h3 class="connect">LET’S CONNECT</h3>
+      <div class="icon-container">
+        <a class="social-icons" href="#"><img src="/images/Twitter.png" alt="Twitter"></a>
+        <a class="social-icons" href="#"><img src="/images/Linkedin-icon.png" alt="Linkedin"></a>
+        <a class="social-icons" href="#"><img src="/images/Medium.png" alt="Medium"></a>
+        <a class="social-icons" href="#"><img src="/images/Github.png" alt="Github"></a>
+        <a class="social-icons" href="#"><img src="/images/Angellist.png" alt="Angellist"></a>
```

Comment:
- [ ] Hi you have designed this section really well 👏. Although your social media icon images are not displaying. the issue is you are defining the `src` attribute using an absolute path. you want the path to start with `./` so that it is relative to the file location. ensure that your image links are working by setting the `src` attribute correctly thanks 👍
![image](https://user-images.githubusercontent.com/18730106/144293198-40897f05-5b88-458e-8afe-e8cfd729004e.png)